ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-routing.j2

```diff
@@ -1,24 +1,22 @@
 {# eos - IP Routing #}
-{% if ip_routing is defined and ip_routing is not none %}
-{%     if ip_routing == true %}
+{% if ip_routing is arista.avd.defined %}
```

Comment:
Both if statement can be merged as `arista.avd.defined` does a deep test

Reply:
this check is redundant and should be removed

Reply:
Changing this is a change in the behaviour of the code. Today we first check that ip_routing is defined. If it is not defined, we don't write anything. With the requested change, if ip_routing is not defined we will write "no ip routing" in the configuration.

Reply:
So then do first `if` as it is in line 3 and replace the `else` in line 6 with `elif ip_routing is arista.avd.defined(false)`. We have similar logic elsewhere and it makes it super easy to read.